data/jobs.txt

```diff
@@ -1573,3 +1573,192 @@ mission "Large Rush Delivery [3]"
 		payment
 		payment 42000
 		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+
+mission "Recycle garbage"
```

Comment:
This seems like a fair mission. The original Escape Velocity also had "Dump Garbage" missions where you dumped them on uninhabited worlds. I don't think Endless Sky allows you to land on such worlds, but perhaps that's a similar mission you could add, perhaps with some backwater planet agreeing to take their trash. Not all waste is recyclable after all.